docs/ruleset_functions/is_zone_mechanically_heated_and_not_cooled.md

```diff
@@ -0,0 +1,34 @@
+# is_zone_mechanically_heated_and_not_cooled
+**Schema Version:** 0.0.23  
+
+**Description:** determines whether a zone is mechanically heated, but not cooled.  DOES NOT CHECK FOR TRANSFER AIR
+
+**Inputs:**
+- **RMD**
+- **zone**
+
+**Returns:**  
+- **result**: a boolean - TRUE or FALSE
+ 
+**Function Call:**
+- **get_list_hvac_systems_associated_with_zone**
+
+## Logic:
+- set result to FALSE: `result = FALSE`
+- get the hvac systems serving the zone by using the get_list_hvac_systems_associated_with_zone function: `list_hvac_systems = get_list_hvac_systems_associated_with_zone(RMD,zone)`
+	- loop through each HeatingVentilatingAirconditionsSystem system: `for system in list_hvac_systems:`
+		- check for a heating system: `if system.heating_system != "Null":`
+			- make sure the heating system type is not equal to None: `if system.heating_system.heating_system_type != NONE:`
+				- look for a cooling system: `if system.cooling_system == "Null":`
```

Comment:
@KarenWGard do we also need to check the terminal heating system type? I believe we're defining PTACs and FPFCUs using the terminal.

Reply:
look for a cooling system: `if system.cooling_system == "Null":`
this line could be look for NULL or None.

Reply:
@supriyagoel - yes, we definitely need to look in terminals.  I updated the function to go through terminals.

Reply:
@weilixu - can the following logic handle on one line, or is it better to do the first check first and then the second?

`if system.cooling_system == NULL or system.cooling_system == NONE`

Reply:
In code, it can be handled in one line.
In RDS, I suggest to lay out the logics for public reference.